.github/workflows/ci_windows.yaml

```diff
@@ -44,26 +44,13 @@ jobs:
 
       - name: Install dependencies from vcpkg
         run: |
-          # tell vcpkg to only build Release variants of the dependencies
```

Comment:
Why did you change it?

Reply:
vcpkg upstream has included this functionality so we don't need to do it ourselves.